src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/Compiler.cs

```diff
@@ -43,9 +43,9 @@ public Sort(int sortkey, string xmllang, XmlDataType datatype, XmlSortOrder xmlo
     internal enum ScriptingLanguage
     {
         JScript,
-#if !FEATURE_PAL // visualbasic
+#if !TARGET_UNIX // visualbasic
```

Comment:
`FEATURE_PAL` was not defined at all, even for Unix in this part of the build before. This ifdef should be deleted and what's under it kept